### DISCUSSION TITLE
Flowable dmn-table editor alternative

### DISCUSSION THREAD

**Discussion by billerby** [valb3r/flowable-bpmn-intellij-plugin#374] [General]
Hi!

As of version 7.x of Flowable Open Source the ui-tools have been removed. Flowable are from this version only offering a cloud-based modeler for editing bpmn-models and dmn-tables. We have been using flowable extensively for seven years in our organization and have multiple applications running built upon the process engine. We are now investigating what needs to be done to stay open source and upgrade to version 7.x. Apart from developing replacements for the Admin-app, the IDM-app and the content-engine (thats also has been removed), we need to address the modeling. Since we do not want to upload our buisiness models to the cloud, we are looking into this plugin and will probably go for the commercial version (polybpm). However we still need a way to edit our dmn-tables and get it in to our development flow in a nice way (checking in to source control etc).

We have found some dmn-editors but none seems to be able to import the models flowable exports in a nice way. We also noticed some discrepancies when it comes to exporting string values and quotation using the bpmn.io/dmn editor.

Just wanted to check if any of you know of any dmn-editors that works ootb against the flowable-engine?

Regards!
/Erik

**Comment by valb3r** [comment]
@billerby Thanks for the input, 

> Just wanted to check if any of you know of any dmn-editors that works ootb against the flowable-engine?

Nope, due to its own specific dialect :( That's the problem with all BPMN/DMN etc. systems

I can suggest opening an issue at https://github.com/GDNext-GmbH/polybpmn regarding DMN support as we are at phase of 'next steps' discussion for it, so it might be a feature we can implement. 

Also, 
> However we still need a way to edit our dmn-tables and get it in to our development flow in a nice way (checking in to source control etc).

If there is a need for such editor and your business is built on that, I can suggest cooperating with us (GDNext GmbH) so that we can develop custom plugin exclusively for your needs

**Reply by billerby** [reply]
Thanks! I have now created a feature request, I will bring up you offer to possibly develop a custom plugin for us with our sponsors.

/Erik
